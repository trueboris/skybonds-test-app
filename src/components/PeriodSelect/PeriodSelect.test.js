import React from 'react';
import renderer from 'react-test-renderer';
import { PeriodSelectView } from './index';

it('renders correctly', () => {
  const element = renderer.create(<PeriodSelectView />).toJSON();

  expect(element).toMatchSnapshot();
});
