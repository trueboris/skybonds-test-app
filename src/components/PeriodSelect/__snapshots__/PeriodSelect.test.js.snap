// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-radio-group ant-radio-group-outline ant-radio-group-large"
>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value="week"
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      week
    </span>
  </label>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value="month"
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      month
    </span>
  </label>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value="quarter"
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      quarter
    </span>
  </label>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value="year"
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      year
    </span>
  </label>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value="max"
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      max
    </span>
  </label>
</div>
`;
